docs/self-managed/platform-deployment/helm-kubernetes/guides/aws-marketplace.md

```diff
@@ -274,11 +274,12 @@ identity:
   contextPath: "/identity"
   fullURL: "https://$CAMUNDA_HOSTNAME/identity"
 
-zeebe-gateway:
+zeebeGateway:
+  contextPath: "/zeebe"
   ingress:
-    enabled: true
-    className: nginx
-    host: "$CAMUNDA_HOSTNAME"
+    grpc:
```

Comment:
:thought_balloon: The idea here is that the global ingress already has the route for the REST API, so you only need the gRPC specific ingress.